ql/ql/src/codeql_ql/ast/internal/Module.qll

```diff
@@ -288,14 +313,6 @@ module ModConsistency {
         .regexpMatch(".*/(test|examples|ql-training|recorded-call-graph-metrics)/.*")
   }
 
-  query predicate noResolveModuleExpr(ModuleExpr me) {
```

Comment:
I'm slightly puzzled as to why this was just removed. Does it not provide any useful information now?

Reply:
It is because a `ModuleExpr` is now just a special `TypeRef`.   
And we test that all `TypeRef`s resolve to a type in `Type.qll`.